sdcm/ycsb_thread.py

```diff
@@ -197,11 +198,17 @@ def parse_final_output(result):
         output = {k: str(v) for k, v in output.items()}
         return output
 
+    def db_node_to_query(self, loader):
+        """Select DB node in the same region as loader node to query"""
+        if self.params.get("region_aware_loader", False):
+            return random.choice([db_node for db_node in self.node_list if db_node.region == loader.region]).ip_address
```

Comment:
I think we should use the seed nodes only... 
and I think we don't need a flag for it...

Reply:
1) why seeds only?
2) flag for what?

Reply:
seed only, since other nodes can down, i.e. we don't do nemesis on seeds.
I've meant we don't need the `region_aware_loader` flag, since it's based on how we configured the loaders number

Reply:
agree and generally agree regarding seeds, but the question is: are seeds selected in each DC(i.e. can I be sure that in each region we will have at least one seed)?

Reply:
- unfortunately, set_seeds doesn't support multi-region...(added a task to support it) so meanwhile I will filter those that nemesis is not running on them.
- I thought about `region_aware_loader` flag again, and in my opinion, for testing purposes, we should support not running on local DC so I am keeping the flag. @roydahan WUT?